docs/proposals/keeping-transient-pod-name-spaces-alive.md

```diff
@@ -0,0 +1,54 @@
+# Keeping transient pod name spaces alive
+
+## Introduction
+The goal of this proposal is to give administrators options to keep namespaces and therefore APB pods after the execution. The use cases for this feature are demos and debugging APBs.
```

Comment:
There's also a use-case for production cluster operators; I don't think this is limited to demo's or debugging APBs. APB's could certainly fail under production scenarios, and having nothing to examine after something unexpected takes place is undesirable.

Reply:
how do we help the production operator be able to debug things like this? would the workflow be if things fail, reconfigure the broker to keep the namespaces around, and retry?